hips/utils/tests/test_healpix.py

```diff
@@ -22,3 +24,17 @@ def test_boundaries():
     radec_precomp = [[264.375,  258.75,  264.375,  270.],
                      [-24.624318,  -30.,  -35.685335,  -30.]]
     assert_allclose([radec.ra, radec.dec], radec_precomp)
+
+def test_compute_healpix_pixel_indices():
+    order = 3
+    nside = hp.order2nside(order)
```

Comment:
I would suggest to move this down to where you use it, i.e. above the `compute_healpix_pixel_indices` line.
Also: you don't need the `order` local variable, i.e. just spare a line and write:
```
nside = hp.order2nside(order=3)
```